cli/container.go

```diff
@@ -62,6 +62,17 @@ func startContainer(verbose bool, config *container.Config, hostConfig *containe
 	return resp.ID, nil
 }
 
+func processVolumes(volumes []string) []string {
+	newVols := make([]string, 0)
+	for _, v := range volumes {
+		if v != "" {
```

Comment:
I agree that we should check for malformed user input by checking if an empty string was passed in, but then we should probably yield an error instead of "correcting" it on our own or what do you think?
With the new StringSlice flag type from #46 , this would make even more sense, since the user has to pass in `--volume ""` for this to happen.

For the current codebase of this PR it makes sense, but probably we can remove this after #46 got merged and you rebase your PR upon that :+1:

Reply:
Yeah for sure, I was like "ohhhh this is hacky" doing this to begin with, haha.